### PR TITLE
make nullable first and last name in bundle quotes

### PR DESCRIPTION
**Pull request by meletios**
# Jira Issue: []

## What?
- Connected to https://github.com/HedvigInsurance/underwriter/pull/368

## Why?
- To enable web to call the bundleQuote endpoint without a first and last name for our partners

## Optional checklist
- [ ] Unit tests written
- [ ] Tested locally
- [ ] Codescouted

### CODE REVIEW

File: src/generated/graphql.ts [hunk 1]
```diff
@@ -210,8 +210,8 @@ export type BundledQuote = {
   id: Scalars['ID'];
   currentInsurer?: Maybe<CurrentInsurer>;
   price: MonetaryAmountV2;
-  firstName: Scalars['String'];
-  lastName: Scalars['String'];
```
Comment (cpiehl1) [inline src/generated/graphql.ts:214]: Hmm, did you change this file manually? How come there is no change to the schema?

**Review by mehdielket — APPROVED** [review]
Looks good!

**Review by robinandeer — COMMENTED** [review]
I know we have a discussion in Slack about this but should we really break backwards compatibility if we can instead use different fields where firstname/lastname are optional?

**Comment by sampettersson** [comment]
I wonder what actually happens in prod, if we in practice rarely send null, then it should work even though we don't update the clients, right? So we could just update Codegen behind the scenes, adapt the code and we good?

**Comment by robinandeer** [comment]
I would still prefer making non-breaking changes and deprecating these fields.

My gut feeling is that it wouldn't cause big issues. However, it is still an assumption and web can simply push out a hot fix if we are wrong -- so if apps are okey with it? ;)

**Comment by hedvigoscar** [comment]
I agree with @robinandeer in the general case - breaking changes should be limited only to the very necessary, and even then after a longer deprecation period where all clients have had ample time to migrate to alternative APIs.

In this particular case I'm not opposed to the change, from the perspective of the android-platform, as we never query these fields.

**Comment by mehdielket** [comment]
Same here. I agree with you @robinandeer in principle, but I think in this specific case this will not affect us either, so if we can avoid deprecation all together than that's better

**Comment by robinandeer** [comment]
Okey, it seems I'm outnumbered 😉

Let's go with this solution and test that our assumptions hold up in staging!

**Review by guilhermespopolin — APPROVED** [review]
Let's test it on stage

**Comment by robinandeer** [comment]
Why can't we test in a review app? @guilhermespopolin

**Pull request merged**